container/docker/handler.go

```diff
@@ -160,8 +163,12 @@ func newDockerContainerHandler(
 		fsInfo:             fsInfo,
 		rootFs:             rootFs,
 		rootfsStorageDir:   rootfsStorageDir,
-		fsHandler:          newFsHandler(time.Minute, rootfsStorageDir, otherStorageDir, fsInfo),
 		envs:               make(map[string]string),
+		ignoreMetrics:      ignoreMetrics,
+	}
+
+	if _, ignore := ignoreMetrics[container.DiskUsageMetrics]; !ignore {
```

Comment:
I'm fine with the complex type, but just wanted to clarify my suggestion. Using a bool value that's always true works *because* the zero value is a bool (false). I recognize it can be a little ambiguous, but I think it's ok for contained usages. Since the set is used across files, I think your approach is good.